src/org/labkey/filetransfer/view/fileTransferConfig.jsp

```diff
@@ -16,31 +16,26 @@
      * limitations under the License.
      */
 %>
-<%@ page import="org.labkey.api.data.Container" %>
-<%@ page import="org.labkey.api.security.User" %>
-<%@ page import="org.labkey.api.view.template.ClientDependencies" %>
-<%@ page import="org.labkey.filetransfer.FileTransferConfigForm" %>
+
 <%@ page import="org.labkey.api.view.JspView" %>
-<%@ page import="org.labkey.api.view.HttpView" %>
+<%@ page import="org.labkey.api.view.template.ClientDependencies" %>
+<%@ page import="org.labkey.filetransfer.view.FileTransferConfigForm" %>
 <%@ page extends="org.labkey.api.jsp.JspBase" %>
 <%!
     @Override
     public void addClientDependencies(ClientDependencies dependencies)
     {
-        dependencies.add("Ext4");
```

Comment:
Good question.  I'm not sure why that happened, but it's been added back in now.